models/app.model.js

```diff
@@ -20,4 +20,11 @@ exports.checkEndpoints = () => {
       .catch(error => {
         console.error(error);
```

Comment:
you want to invoke next here with your error so that it can go to the next piece of error handling middleware to be handled